web-components/src/api/folksonomy.ts

```diff
@@ -16,6 +17,118 @@ const getApiUrl = (path: string) => {
   return `${folksonomyConfiguration.getItem("apiUrl")}${path}`
 }
 
+/**
+ * Get stored token from localStorage
+ * @returns {string | null}
+ */
+function getStoredToken(): string | null {
+  return localStorage.getItem('bearer')
```

Comment:
We should name it folksonomy-bearer-token

It's more easy to understand, and there is less chance of name clash with other libraries / web-components.

Reply:
the change must be propagated where it is used.
I would even say to use a constant for the name, to avoid typos and so no.